drivers/timer/Kconfig.stm32_lptim

```diff
@@ -4,8 +4,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 menuconfig STM32_LPTIM_TIMER
-	bool "STM32 Low Power Timer"
-	depends on (SOC_SERIES_STM32L4X || SOC_SERIES_STM32WBX)
+	bool "STM32 Low Power Timer [EXPERIMENTAL]"
+	depends on SOC_SERIES_STM32L4X && DEVICE_POWER_MANAGEMENT
```

Comment:
Minor nitpick: I don't fully see the relation of why the LPTIM support needs to depend on DEVICE_POWER_MANAGEMENT.

Reply:
My goal is to link lptim to low power mode. 
This let kernel timer unchanged for most of the usecase.
The lptim is able to preserve the kernel tick counter during low power sleep mode : only in this case is the lptim configured to keep the kernel timeout. 
Refering to  power: stm32l4: support of the Low Power Mode #19026

Reply:
Understood. In that case shouldn't it be `SYS_POWER_MANAGEMENT` instead of `DEVICE_POWER_MANAGEMENT`?